src/phoneInput.tsx

```diff
@@ -93,17 +105,17 @@ export class PhoneInput extends React.Component<PhoneInputProps> {
     onChange && onChange(alpha2, national)
   }
 
-  handleClick = (event: any) => {
+  handleClick: React.MouseEventHandler<HTMLInputElement> = (event) => {
```

Comment:
@DanielFuchs2 this click event is on `<Button />` element so it should be `React.MouseEventHandler<HTMLElement>`